docs/mine/lotus/miner-wallets.md

```diff
@@ -1,145 +1,149 @@
 
-- Changing the owner or the worker address in the _miner actor_
-- Withdrawing balance from the _miner actor_
-- Submit _WindowPoSts_ (**unless _control addresses_ are defined and have enough balance**, see below)
+- Changing the owner or the worker address in the _miner actor_.
+- Withdrawing balance from the _miner actor_.
+- Submit _WindowPoSts_, **unless _control addresses_ are defined and have enough balance** (continued below).
 
-The wallet associated to the _owner address_ is designed to be kept offline as a cold wallet, given that it should not be used often and it is critical to safeguard the miner's funds. In production environments, we strongly recommend using separate _owner_ and _worker_ addresses.
+The wallet associated with the _owner address_ is designed to be kept offline as a cold wallet since it should not be used often. It is critical to safeguard the miner's funds. In production environments, we strongly recommend using separate _owner_ and _worker_ addresses.
 
 The owner address can be updated with the following command:
 
 ```sh
 lotus-miner actor set-owner --really-do-it <address>
 ```
 
-The old and the new address must be available in the Lotus node. For the new address, you can [create a new one or import an existing one](../../get-started/lotus/send-and-receive-fil.md).
+The old and the new address must be available in the Lotus node. You can [create a new address or import an existing one](../../get-started/lotus/send-and-receive-fil.md).
 
 ## The worker address
 
 The _worker address_ is used to send and pay for day-to-day operations performed by the miner:
 
-- Initializing the miner on the chain
-- Changing the miner peer id or the multiaddresses
-- Interacting with market and payment channel actors
-- Signing new blocks
+- Initializing the miner on the chain.
+- Changing the miner peer id or the multiaddresses.
+- Interacting with market and payment channel actors.
+- Signing new blocks.
 - Submitting proofs, declaring faults. _WindowPoSts_ are submitted using the _worker address_ if:
-  - Control addresses are not defined or do not have enough balance
-  - The _owner address_ does not have enough balance
+  - Control addresses are not defined or do not have enough balance.
+  - The _owner address_ does not have enough balance.
 
-Thus, unlike the _owner wallet_, the _worker wallet_ must necessarily be a _hot_ wallet. The Lotus Miner will trigger all the necessary transactions using the Lotus Node to which it is connected. The _worker address_ must have enough funds to pay for the day-to-day operations of the miner, including initialization.
+Unlike the _owner wallet_, the _worker wallet_ must be a _hot_ wallet. The Lotus Miner will trigger all the necessary transactions using the Lotus node to which it is connected. The _worker address_ must have enough funds to pay for the day-to-day operations of the miner, including initialization.
```

Comment:
@hsanjuan, again, I'm confused by this terminology. Hot-wallets exist, but if there's only 1 wallet per node, then how can we have several wallets?

Reply:
How about something like: `Unlike the _owner  address_, the address set as the miner's _worker address_ should be part of the Lotus wallet and accessible by the miner.`

Reply:
```suggestion
Unlike the _owner address_, the address set as the miner's _worker address_ should be part of the Lotus local wallet and accessible by the miner. The Lotus Miner will trigger all the necessary transactions using the Lotus node to which it is connected. The _worker address_ must have enough funds to pay for the day-to-day operations of the miner, including initialization.
```